src/Commands/stubs/event.stub

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace $NAMESPACE$;
+
+use $DUMMYNAMESPACE$\Event;
```

Comment:
Why is this line needed ? What's dummy namespace ? Events by default don't really need to extend anything.